consensus/src/alerts/mod.rs

```diff
@@ -406,18 +406,13 @@ impl<'a, H: Hasher, D: Data, MK: MultiKeychain> Alerter<'a, H, D, MK> {
     }
 }
 
-pub type MessagesForNetwork<H, D, MK> = (
-    AlertMessage<H, D, <MK as Keychain>::Signature, <MK as MultiKeychain>::PartialMultisignature>,
-    Recipient,
-);
-
-pub type MessagesFromNetwork<H, D, MK> =
+pub type NetworkMessages<H, D, MK> =
```

Comment:
Agh, sorry I keep being nitpicky, but this probably should be singular not plural.

Reply:
Naming is important, changed.